src/framework/COScript.m

```diff
@@ -405,6 +405,56 @@ - (void)deleteObjectWithName:(NSString*)name {
     [_mochaRuntime removeObjectWithName:name];
 }
 
+# pragma mark - require
+
+- (NSURL*)resolveModule:(NSString *)module currentURL:(NSURL*)currentURL {
```

Comment:
I'd personally appreciate a bit of docs here, in particular noting what format `module` is in/where it comes from.